### PR TITLE
move errorBoundary outside Router

### PR DESCRIPTION
**Pull request by mydiemho**
this broke navigation render, settings doesn't load until
refresh

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -52,24 +52,26 @@ class App extends React.Component<IAppProps> {
...
-                            <MainContentRouter />
+            <Fragment>
+                <Alert title={errorTitle}
+                    message={errorMessage}
+                    closeButtonColor="info"
+                    show={showError}
+                    onClose={this.props.actions.clearError}
+                />
+                <ErrorBoundary>
+                    <Router>
```
Comment (wbreza) [inline src/App.tsx]: Why are we introducing another `Router`? We are already using `Router` inside of `<MainContentRouter />`

Comment (tmiller-msft) [reply src/App.tsx]: I'm not sure why it's there, but it's been there for a long time, wasn't introduced as part of this change.

Comment (mydiemho) [reply src/App.tsx]: I don't see Router being use in MainContentRouter, is `<Route`> the same as `<Router>`

Comment (wbreza) [reply src/App.tsx]: You guys are right - sorry 👍

**Pull request merged**